docs/doxygen.ini

```diff
@@ -1,4 +1,4 @@
-# Doxyfile 1.9.6
+# Doxyfile 1.10.0
```

Comment:
All these changes are done automatically (by running `doxygen -u`) - fixes some of warnings during docs generation (various deprecations).